tests/sentry/eventstore/test_models.py

```diff
@@ -26,7 +26,7 @@ def test_pickling_compat(self):
 
         # When we pickle an event we need to make sure our canonical code
         # does not appear here or it breaks old workers.
-        data = pickle.dumps(event, protocol=2)
+        data = pickle.dumps(event, protocol=5)
```

Comment:
Why does this one change?

Reply:
ooh leftover of me testing to see if changing the protocol broke anything, do we want a different tests that explicitly passes in the protocol?

Reply:
hmmm, wonder where we should put that test